lib/trino-filesystem-gcs/src/test/java/io/trino/filesystem/gcs/TestGcsFileSystemGcs.java

```diff
@@ -28,4 +34,16 @@ void setup()
     {
         initialize(getRequiredEnvironmentVariable("GCP_CREDENTIALS_KEY"));
     }
+
+    @Test
+    void testCreateFileRetry()
```

Comment:
How does this test verifies retries take place?
If it is not obvious how the test works,  is it worth documenting?

Reply:
Sure will do. Just for context: this fails consistently without the retries. Can repro locally as well.